@client/avatar.coffee

```diff
@@ -40,7 +40,7 @@ window.AvatarPopover = ReactiveComponent
     {user, anon, opinion} = @props
     user = fetch user
 
-    anonymous = user.key != fetch('/current_user').user && (customization('anonymize_everything') || anon)      
```

Comment:
I don't notice any cases where I need to see my own avatar to find my own anonymous content.

-     For comments, there is an edit-button by my own comments
-     For reasons, there is a button to view all my own reasons
-     For proposals, there is an edit-button by my own proposals

Reply:
Good point! But those don't show up after a forum has been frozen. And it is nice to be able to scan for your icon. 

That said, showing the icon is problematic because people might be scared that their content was not actually posted anonymously. So I'd be in favor of removing user.key != fetch('/current_user').user from the conditional.